src/spritekit.cs

```diff
@@ -1954,28 +1991,28 @@ partial interface SKAction : NSCoding, NSCopying {
 		
 		// These are in a category
 		[Static, Export ("moveByX:y:duration:")]
-		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double sec);
+		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double duration);
```

Comment:
But I still consider this is a good one, it really helps to better understand the API and even more if we add the measurement unit to the parameter name